### DISCUSSION TITLE
Parse error with PHP 7.4.33 starting from version 7.5.6

### DISCUSSION THREAD

**Discussion by albertdick** [deployphp/deployer#3987] [Bugs] [answered]
### Deployer Version

v7.5.6

### Target OS

Ubuntu 22.04.4, Windows 10 Pro

### Which PHP version are you using?

PHP 7.4

### Steps to reproduce

`deployer.phar` stop working from version 7.5.6 using PHP 7.4.33, triggering a parse error:

`PHP Parse error:  syntax error, unexpected ')', expecting variable (T_VARIABLE) in phar:///home/ubuntu/bin/dep/vendor/ringcentral/psr7/src/functions.php on line 355`

**Comment by yugeon** [comment]
confirm the same bug, not fixed in 7.5.7, 7.5.8
```bash
> php deployer.phar

Parse error: syntax error, unexpected ')', expecting variable (T_VARIABLE) in phar://H:/work/PHP/test/deployer.phar/vendor/ringcentral/psr7/src/functions.php on line 355
PHP Parse error:  syntax error, unexpected ')', expecting variable (T_VARIABLE) in phar://H:/work/PHP/test/deployer.phar/vendor/ringcentral/psr7/src/functions.php on line 355
Script php deployer.phar handling the dep event returned with error code 255
```

```bash
> php -v
PHP 7.4.33 (cli) (built: Nov  2 2022 15:06:48) ( NTS Visual C++ 2017 x64 )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
```

**Comment by sfroemkenjw** [comment]
In all available tags of https://github.com/ringcentral/psr7/blob/1.3.0/src/functions.php I can not find the comma behind last argument (IMO only available since 8.*) of that function. Seems, the issue comes in with a CGL automatism.

**Comment by Mabahe** [comment]
Yes, this discussion is a dupe to discussion https://github.com/deployphp/deployer/discussions/3974 

Yes, you can not find this in the related lib repo. In v7 vendor dir is committed in the repo and was changed by cs-fixer. Check comment https://github.com/deployphp/deployer/discussions/3974#discussioncomment-11779579

**Comment by mikestreety** [comment, marked as answer]
This is now resolved as of [7.5.11](https://github.com/deployphp/deployer/releases/tag/v7.5.11).
